### PR TITLE
ENT-4126: Set phar read only off for PHP

### PR DESCRIPTION
**Pull request by aleksandrychev**
New Export/Import tooling requires phar

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @Lex-2008 can review this?

File: deps-packaging/php/debian/rules [hunk 1]
```diff
@@ -139,6 +139,9 @@ install: build
 	# Set the default timezone for php to UTC
 	sed -ri 's/^(\s|;)*date.timezone\s*=.*/date.timezone = "UTC"/g' $(CURDIR)/debian/tmp$(PREFIX)/httpd/php/lib/php.ini
 
+	# Set the phar readonly Off for php
+    sed -ri 's/^(\s|;)phar.readonly = On/phar.readonly = Off/g' $(CURDIR)/debian/tmp$(PREFIX)/httpd/php/lib/php.ini
```
Comment (Lex-2008) [inline deps-packaging/php/debian/rules]: this file is makefile, so it must be indented with **tab**, not _spaces_

Comment (aleksandrychev) [reply deps-packaging/php/debian/rules]: ok, thanks!

**Comment by craigcomstock** [comment]
you don't need trailing `/g` right? there's only one instance of the search pattern.

**Comment by aleksandrychev** [comment]
>you don't need trailing /g right? there's only one instance of the search pattern.

@craigcomstock yes, right

**Pull request merged**